eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/SubmittedVariantAccessioningServiceTest.java

```diff
@@ -104,7 +165,176 @@ public void sameAccessionsAreReturnedForEquivalentVariants() throws AccessionCou
+        List<SubmittedVariant> variants = Arrays.asList(submittedVariant, newSubmittedVariant, dbsnpSubmittedVariant);
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> submittedVariants = service.getOrCreate(variants);
+
+        long accession = submittedVariants.stream()
+                                          .filter(x -> x.getData().getProjectAccession().equals(PROJECT))
+                                          .collect(Collectors.toList())
+                                          .get(0).getAccession();
+        assertEquals(ACCESSION, accession);
+
+        long accessionDbsnp = submittedVariants.stream()
+                                               .filter(x -> x.getData().getProjectAccession().equals(PROJECT_DBSNP))
+                                               .collect(Collectors.toList())
+                                               .get(0).getAccession();
+        assertEquals(ACCESSION_DBSNP_1, accessionDbsnp);
+
+        assertEquals(3, submittedVariants.size());
+    }
+
+    @UsingDataSet(locations = {"/test-data/submittedVariantEntity.json", "/test-data/dbsnpSubmittedVariantEntity.json"})
+    @Test
+    public void getFromBothRepositories() {
+        List<SubmittedVariant> variants = Arrays.asList(submittedVariant, newSubmittedVariant, dbsnpSubmittedVariant);
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> accessions = service.get(variants);
+        assertEquals(2, accessions.size());
     }
+
+    @UsingDataSet(locations = {"/test-data/submittedVariantEntity.json", "/test-data/dbsnpSubmittedVariantEntity.json"})
+    @Test
+    public void getByAccessionsFromBothRepositories() {
+        List<Long> accessions = Arrays.asList(ACCESSION, ACCESSION_DBSNP_1);
+        List<AccessionWrapper<ISubmittedVariant, String, Long>> submittedVariants = service.getByAccessions(accessions);
+        assertEquals(2, submittedVariants.size());
+    }
+
+    @UsingDataSet(locations = {"/test-data/submittedVariantEntity.json", "/test-data/dbsnpSubmittedVariantEntity.json"})
+    @Test
+    public void getByAccessionAndVersionFromBothRepositories() throws AccessionMergedException,
+            AccessionDoesNotExistException, AccessionDeprecatedException {
+        AccessionWrapper<ISubmittedVariant, String, Long> submittedVariant = service.getByAccessionAndVersion(
+                ACCESSION, 1);
+        assertEquals(this.submittedVariant, submittedVariant.getData());
+
+        AccessionWrapper<ISubmittedVariant, String, Long> dbsnpSubmittedVariant = service.getByAccessionAndVersion(
+                ACCESSION_DBSNP_1, 1);
+        assertEquals(this.dbsnpSubmittedVariant, dbsnpSubmittedVariant.getData());
+    }
+
+    @UsingDataSet(locations = {"/test-data/submittedVariantEntity.json"})
+    @Test
+    public void updateSubmittedVariant() throws AccessionDeprecatedException, AccessionDoesNotExistException,
```

Comment:
We have this test to check that we can update variants, but, can a dbSNP variant be updated? If so, another test is needed.

Reply:
There is a separate test called `updateDbsnpSubmittedVariant`

Reply:
👍